python_examples/simple_parameters.json

```diff
@@ -368,5 +368,32 @@
                                     0.47651, 0.47651, 0.47651, 0.47651,
                                     0.47651, 0.47651, 0.47651, 0.47651,
                                     0.47651]
+    },
+    "household_age_params": {
```

Comment:
Do we need this in the simple sweep? Maybe not worth bothering there and only have it in the more complete spatial one (or add an age-dependant one if you want - I can help on this as was going to write some age dependant plotting)

Reply:
I think Lizzi added this so perhaps worth discussing with her first but I agree that this is probably best in a more advanced example simulation